src/pe_source/data/pe_db/db_query.py

```diff
@@ -97,33 +97,63 @@ def get_data_source_uid(source):
 def insert_sixgill_alerts(df):
     """Insert sixgill alert data."""
     conn = connect()
-    df = df[
-        [
-            "alert_name",
-            "content",
-            "date",
-            "sixgill_id",
-            "read",
-            "severity",
-            "site",
-            "threat_level",
-            "threats",
-            "title",
-            "user_id",
-            "category",
-            "lang",
-            "organizations_uid",
-            "data_source_uid",
+    try:
+        df = df[
+            [
+                "alert_name",
+                "content",
+                "date",
+                "sixgill_id",
+                "read",
+                "severity",
+                "site",
+                "threat_level",
+                "threats",
+                "title",
+                "user_id",
+                "category",
+                "lang",
+                "organizations_uid",
+                "data_source_uid",
+                "content_snip",
+                "asset_mentioned",
+                "asset_type",
+            ]
+        ]
+    except Exception as e:
+        logging.error(e)
+        df = df[
+            [
+                "alert_name",
+                "content",
+                "date",
+                "sixgill_id",
+                "read",
+                "severity",
+                "site",
+                "threat_level",
+                "threats",
+                "title",
+                "user_id",
+                "organizations_uid",
+                "data_source_uid",
+                "content_snip",
+                "asset_mentioned",
+                "asset_type",
+            ]
```

Comment:
This try/except block could use a comment with a bit more info about what's going on here.

Reply:
resolved: [036b1d9](https://github.com/cisagov/pe-reports/pull/193/commits/036b1d9ca88dc6966e5914e5ae658140e7ec568c)

Reply:
Reminder to please keep unrelated changes in separate commits- it makes PRs much easier to re-review.

Also, it's helpful to mention that in this particular case, the conversation was resolved by removing the confusing block of code, not by adding a comment.